api/api-rules/README.md

```diff
@@ -12,11 +12,20 @@ e.g.
 
  * ***API rule violation: names_match,k8s.io/api/core/v1,Event,ReportingController***
 
+And the violation list is sorted alphabetically in each of the \<RULE\>, \<PACKAGE\>, \<TYPE\>, \<FIELD\> levels.
+
 Make rule returns an error when the newly generated violation report differs from this
 checked-in violation report. If a new API rule violation is detected, please fix
 the API Go source file to pass the API rule check. **The entries in the checked-in
 violation report should only be removed when existing API rule violation is
 being fixed, but not added.**
 
+If you're removing violations from the exception list, or if there are good
+reasons for new APIs to violate API rules, please update the file using:
+
+ - `make UPDATE_API_KNOWN_VIOLATIONS=true generated_files`
```

Comment:
I think usually we've put the variable after the target, e.g. `make generated_files UPDATE_API_KNOWN_VIOLATIONS=true` but I'm not sure if it makes a difference in practice.